src/components/DpAutocomplete/DpAutocomplete.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+
+import DpAutocomplete from './DpAutocomplete.vue'
+
+const meta: Meta<typeof DpAutocomplete> = {
+    component: DpAutocomplete,
+    title: "Components/DpAutocomplete",
+    argTypes: {
+        'searchChanged': { action: 'searchChanged' }
+    }
+}
+
+export default meta
+type Story = StoryObj<typeof DpAutocomplete>
+
+export const Default: Story = {
+    args: {
+        height: '28px',
+        label: 'label',
+        routeGenerator: function () {},
```

Comment:
this function should return a (empty) string
```suggestion
        routeGenerator: () => '',
```